.travis.yml

```diff
@@ -60,6 +50,7 @@ install:
 
 before_script:
   - yes | ./scripts/resetdb.sh
+  - rvm get head # https://github.com/travis-ci/travis-ci/issues/6307
```

Comment:
(Drive-by): Isn't this only useful for OSX builds?  (Sorry if I missed something, I skimmed the issue rather quickly.)

Reply:
oops, detritus from a previous attempt -- I'll remove